pkg/features/kube_features.go

```diff
@@ -226,8 +227,9 @@ func init() {
 // To add a new feature, define a key for it above and add it here. The features will be
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
-	AppArmor:                                    {Default: true, PreRelease: utilfeature.Beta},
-	DynamicKubeletConfig:                        {Default: false, PreRelease: utilfeature.Alpha},
+	AppArmor:             {Default: true, PreRelease: utilfeature.Beta},
+	DynamicKubeletConfig: {Default: false, PreRelease: utilfeature.Alpha},
+	// KubeletConfigFile is now a no-op gate on the path to removal.
```

Comment:
minimize diff on the other lines for history's sake, omit comment or put at the end of the line

Reply:
done